apps/demo/partials/header.hbs

```diff
@@ -1,8 +1,8 @@
-<header class='bg-inverted text-secondary-inverted py-1000 py-tablet-1100'>
+<header class='spirit-theme-light-inverted bg-primary text-secondary py-1200 py-tablet-1400'>
   <div class='Container'>
 
-    <h1 class='typography-heading-xlarge-text'>Spirit Design System</h1>
```

Comment:
Hmm, do we have a migration guide for the text styles? 🤔

Reply:
Not yet, feel free to create one 🙃